system-x/services/filesystem/src/main/java/software/tnb/filesystem/resource/openshift/OpenShiftFileSystem.java

```diff
@@ -0,0 +1,90 @@
+package software.tnb.filesystem.resource.openshift;
+
+import software.tnb.common.config.OpenshiftConfiguration;
+import software.tnb.common.deployment.OpenshiftDeployable;
+import software.tnb.common.openshift.OpenshiftClient;
+import software.tnb.filesystem.service.FileSystem;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import com.google.auto.service.AutoService;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import cz.xtf.core.openshift.OpenShiftWaiters;
+
+@AutoService(FileSystem.class)
+public class OpenShiftFileSystem extends FileSystem implements OpenshiftDeployable {
```

Comment:
I created a PR https://github.com/tnb-software/TNB/pull/121 so that this doesn't need to implement `OpenshiftDeployable`